src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -194,7 +193,7 @@ class boss_nazan : public CreatureScript
                 uint32 Fly_Timer;
                 uint32 Turn_Timer;
                 bool flight;
-                ObjectGuid VazrudenGUID;
+			    Creature* Vazruden;  //just i don't know how to use guid to start attack,so I change to Creature*, pls help me.
```

Comment:
Use `ObjectAccessor::GetCreature`. Never store `Creature*` across ticks in scripts.